readme.md

```diff
@@ -29,6 +29,8 @@ If command line argument `-f|--not-create-folder` passed, the mp3 file will be d
 The script creates file `downloads.txt` along with downloaded mp3 files.
 This file intended to store youtube IDs of already downloaded files, do not remove it if you're planning to interrupt/resume your download, and vise versa, remove it if you want to download some audious that were downloaded before.
 
+It's also possible to pass a list of urls inside text document `.txt`, where each line is a new video/playlist url. This case if argument `-f` wasn't passed, the folder name will be the same as an input file name.
```

Comment:
It's also possible to pass a list of urls inside text document `.txt`, where each line is a video or playlist url. 

In this case if the argument `-f` was not passed, the folder name will be the same as a folder of the input file.